Makefile

```diff
@@ -45,7 +45,7 @@ all: helm
 
 ##@ Build
 
-helm: helm-keylime ## Builds all helm charts
+helm: helm-build ## Builds all helm charts
```

Comment:
good catch... looks like someone renamed the `helm-keylime` target to `helm-build` without adjusting this up here :+1: